pkg/controller/deployment_rollback.go

```diff
@@ -120,3 +129,97 @@ func (dc *controller) updateMachineDeploymentAndClearRollbackTo(d *v1alpha1.Mach
 	_, err := dc.controlMachineClient.MachineDeployments(d.Namespace).Update(d)
 	return err
 }
+
+// removeTaintNodesBackingMachineSets removes taints from all nodes backing the machineSets
+func (dc *controller) removeTaintNodesBackingMachineSets(machineSet *v1alpha1.MachineSet) error {
```

Comment:
Can we please make this method more generic? 
- ATM PNS taint seems to be hard-coded, we could get PNS as parameter and use same method for diff taints as well.

Also if this function plans to entertain only one MachineSet than can we please name  it as "MachineSet" at the end-singular. 
wdyt, does it makes sense?